### DISCUSSION TITLE
Block clickable hyperlinks

### DISCUSSION THREAD

**Discussion by Daedemons** [sumatrapdfreader/sumatrapdf#4509] [General]
Hi

There is clickable hyperlink in some PDFs. It may be watermark. When you accidentally click on the watermark, you will be redirected to the external webpage. sometimes the watermark may occupy one-third area of the single page. is it convenint to block hyperlinks that can redirect to external webpages.

**Comment by GitHubRulesOK** [comment]
Hmm Hyperlinks come in several flavours but it is common for PDF readers to read "Plaint text" URLs as an active link same as here https://example.com thus Acrobat does have an optional setting to over-ride that. So too does SumatraPDF by means of restrictions.ini

See second item listed https://www.sumatrapdfreader.org/docs/Configure-for-restricted-use

**Reply by Daedemons** [reply]
Thx, it works.
